src/foam/dao/ManyToManyRelationshipDAO.js

```diff
@@ -25,20 +25,30 @@ foam.CLASS({
   documentation: 'Adapts a DAO based on a *:* Relationship.',
 
   properties: [
-    'junctionProperty',
+    'junctionProperty', // TODO(markdittmer): Is this needed?
```

Comment:
Right. This one makes sense to me know; it's needed in the `select()` implementation.